docs/src/main/asciidoc/Application_Development_Guide.adoc

```diff
@@ -0,0 +1,29 @@
+include::_app-dev-guide/attributes.adoc[]
+
+:revnumber: {version}
+:revdate: {localdate}
+:toc: macro
+:toclevels: 2
+:toc-title: {GettingStartedDevelopingApplications}
+:doctype: book
+:icons: font
+:source-highlighter: coderay
+:wildflyVersion: 14
+:sectlinks:
+:idseparator: -
+:include-depth: 2
+
+:appication_development:
+
+= {GettingStartedDevelopingApplications}
```

Comment:
An intro paragraph or two would be good, so people can understand the overall structure and decide how to absorb the info if they don't want to absorb it step by step.

Reply:
I agree, @emmartins.

The reality is most people who are going to look at this document are not going to need explanation of things like how to set up a maven project. If they come to the page and it just jumps into getting into how to set up a maven project with nothing (beyond the TOC) to give them the sense that there's a lot of good content, they are likely to just quickly move on.

Something like

"This is a guide that aims to cover how to develop, from scratch, test and deploy applications for WildFly. We start from the very beginning. Use the TOC to jump ahead if you are already familiar with some areas."

Reply:
Ok with me.

Reply:
Thanks, for the suggestions added the suggested intro with some modifications:

"This guide covers how to develop, from scratch, test, and deploy applications on WildFly. If you are already familiar with some areas, use the table of contents to jump ahead."